CONTRIBUTING.md

```diff
@@ -90,87 +99,101 @@ You must also configure the following environment variables before running the t
+  export AWS_VPC_CIDR_BLOCK=<YOUR_VPC_CIDR_BLOCK>
+  export AWS_REGION=<YOUR_REGION>
+  export AWS_SUBNET_ID=<YOUR_SUBNET_ID>
+  export AWS_SECURITY_GROUP_ID=<YOUR_SECURITY_GROUP_ID>
   ```
-~> **Notice:** For more information about the Network Peering resource, see: https://docs.atlas.mongodb.com/reference/api/vpc/
+
+~> **Notice:** For more information about the Network Peering resource, see: <https://docs.atlas.mongodb.com/reference/api/vpc/>
 
 - For `Encryption at Rest` resource configuration:
+
   ```bash
-  $ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
-  $ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
-  $ export AWS_CUSTOMER_MASTER_KEY_ID=<YOUR_CUSTOMER_MASTER_KEY_ID>
-  $ export AWS_REGION=<YOUR_REGION>
-
-  $ export AWS_ACCESS_KEY_ID_UPDATED=<YOUR_ACCESS_KEY_ID_UPDATED>
-  $ export AWS_SECRET_ACCESS_KEY_UPDATED=<YOUR_SECRET_ACCESS_KEY_UPDATED>
-  $ export AWS_CUSTOMER_MASTER_KEY_ID_UPDATED=<YOUR_CUSTOMER_MASTER_KEY_ID_UPDATED>
-  $ export AWS_REGION_UPDATED=<YOUR_REGION_UPDATED>
+  export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+  export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+  export AWS_CUSTOMER_MASTER_KEY_ID=<YOUR_CUSTOMER_MASTER_KEY_ID>
+  export AWS_REGION=<YOUR_REGION>
+
+  export AWS_ACCESS_KEY_ID_UPDATED=<YOUR_ACCESS_KEY_ID_UPDATED>
+  export AWS_SECRET_ACCESS_KEY_UPDATED=<YOUR_SECRET_ACCESS_KEY_UPDATED>
+  export AWS_CUSTOMER_MASTER_KEY_ID_UPDATED=<YOUR_CUSTOMER_MASTER_KEY_ID_UPDATED>
+  export AWS_REGION_UPDATED=<YOUR_REGION_UPDATED>
   ```
-~> **Notice:** For more information about the Encryption at Rest resource, see: https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/
+
+~> **Notice:** For more information about the Encryption at Rest resource, see: <https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/>
 
 - For `Private Endpoint Link` resource configuration:
+
   ```bash
-  $ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
-  $ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
-  $ export AWS_CUSTOMER_MASTER_KEY_ID=<YOUR_CUSTOMER_MASTER_KEY_ID>
-  $ export AWS_REGION=<YOUR_REGION>
-  $ export AWS_VPC_ID=<YOUR_VPC_ID>
-  $ export AWS_SUBNET_ID=<YOUR_SUBNET_ID>
-  $ export AWS_SECURITY_GROUP_ID=<YOUR_SECURITY_GROUP_ID>
+  export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+  export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+  export AWS_CUSTOMER_MASTER_KEY_ID=<YOUR_CUSTOMER_MASTER_KEY_ID>
+  export AWS_REGION=<YOUR_REGION>
+  export AWS_VPC_ID=<YOUR_VPC_ID>
+  export AWS_SUBNET_ID=<YOUR_SUBNET_ID>
+  export AWS_SECURITY_GROUP_ID=<YOUR_SECURITY_GROUP_ID>
   ```
-~> **Notice:** For more information about the PrivateLink (for AWS only), see: https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/https://docs.atlas.mongodb.com/reference/api/private-endpoint/
+
+~> **Notice:** For more information about the PrivateLink (for AWS only), see: <https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/https://docs.atlas.mongodb.com/reference/api/private-endpoint/>
```

Comment:
in case you want to improve how this renders https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts